Sources/TSCBasic/Path.swift

```diff
@@ -544,11 +546,13 @@ private struct UNIXPath: Path {
 
     init(normalizingAbsolutePath path: String) {
       #if os(Windows)
-        var buffer: [WCHAR] = Array<WCHAR>(repeating: 0, count: Int(MAX_PATH + 1))
-        _ = path.withCString(encodedAs: UTF16.self) {
-            PathCanonicalizeW(&buffer, $0)
+        var result: [WCHAR] = Array<WCHAR>(repeating: 0, count: Int(MAX_PATH + 1))
+        defer { LocalFree(result) }
```

Comment:
The `LocalFree` here is incorrect.  The allocation will be released by the `deinit` of `Array` which will be correct.